sw/airborne/subsystems/stereoprotocol.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * stereoprotocol.h
+ *
+ *  Created on: Sep 23, 2015
+ *      Author: roland
+ */
+
+#ifndef SW_AIRBORNE_SUBSYSTEMS_STEREOPROTOCOL_H_
+#define SW_AIRBORNE_SUBSYSTEMS_STEREOPROTOCOL_H_
+
+#include <inttypes.h>
+#include "mcu_periph/link_device.h"
+struct MsgProperties {
+  uint16_t positionImageStart;
+  uint8_t width;
+  uint8_t height;
+} ;
+typedef struct MsgProperties MsgProperties;
```

Comment:
We don't prefer typedefs.. next to that if you do use a typedef use it before the struct.